core/error_macros.h

```diff
@@ -173,6 +196,18 @@ extern bool _err_error_exists;
 			_err_error_exists = false;                                                                     \
 	}
 
+/** Use this one if there is no sensible fallback, that is, the error is unrecoverable. 
+ */
+
+#define CRASH_COND(m_cond)                                                                                        \
+	{                                                                                                             \
+		if (m_cond) {                                                                                             \
+			_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "FATAL: Condition ' " _STR(m_cond) " ' is true."); \
+			GENERATE_TRAP                                                                                         \
+		} else                                                                                                    \
+			_err_error_exists = false;                                                                            \
```

Comment:
This _err_error_exists is likely not needed, will just set the value unnecesarily all the time (hampering performance) and won't ad anything to when it crashes